### DISCUSSION TITLE
Damaged File for an M1 Macbook pro download

### DISCUSSION THREAD

**Discussion by SECI1** [whs/runekit#30] [Q&A macOS]
Hi everyone,

I have tried downloading the [RuneKit.app.zip](https://github.com/whs/runekit/releases/download/continuous/RuneKit.app.zip) file on my M1 mac. Once it has been downloaded and RuneKit is opened, a message saying '"RuneKit" is damaged and can't be opened. You should move it to the bin.' I have tried to open it again with my privacy settings set to open apps downloaded from anywhere. I am not sure what to do as I am a big Clue scroller but I had to upgrade my previous intel mac to this M1 mac and I can no longer use Alt one or get BootCamp. Does anyone have any advice or tips?
 I am on the latest macOS software (Ventura), and I would really appreciate some advice or solutions.

<img width="281" alt="Screenshot 2022-12-24 at 17 01 46" src="https://user-images.githubusercontent.com/121353884/209445125-a755be8a-3fc9-4b85-beb5-9e273c3e9572.png">

**Comment by madeline-scyphers** [comment]
[This comment here worked for me](https://github.com/whs/runekit/discussions/11#discussioncomment-1244489).
